docs/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: "vkcs"
+page_title: "Provider: VKCS"
+description: |-
+  The VKCS provider is used to interact with VKCS services.
+  The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# VKCS Provider
+
+The VKCS provider is used to interact with
+[VKCS services](https://mcs.mail.ru/). The provider needs
+to be configured with the proper credentials before it can be used.
+
+Use the navigation to the left to read about the available resources.
+
+## Example Usage
+
+Terraform 1.0 and later:
+
+```terraform
+# Configure the vkcs provider
+
+terraform {
+  required_providers {
+    vkcs = {
+      source = "MailRuCloudSolutions/vkcs"
```

Comment:
`MailRuCloudSolutions` -> `vk-cs` ? Also, where we should store releases of provider? Hashiorp registry is not available from RU region.

Reply:
Replaced